subcommand/inject-connect/command.go

```diff
@@ -57,17 +80,35 @@ func (c *Command) init() {
 	c.flagSet.StringVar(&c.flagKeyFile, "tls-key-file", "",
 		"PEM-encoded TLS private key to serve. If blank, will generate random cert.")
 	c.flagSet.StringVar(&c.flagConsulImage, "consul-image", connectinject.DefaultConsulImage,
-		"Docker image for Consul. Defaults to an Consul 1.3.0.")
+		"Docker image for Consul. Defaults to an Consul 1.5.0.")
```

Comment:
```suggestion
		"Docker image for Consul. Defaults to Consul 1.5.0.")
```

Reply:
Updating the default image to be 1.7.0 as it will be needed for namespaces.